packages/runner/lib/sdk/sdk.ts

```diff
@@ -70,6 +71,11 @@ export class NangoActionRunner extends NangoActionBase {
             logger: async (log) => {
                 await this.sendLogToPersist(log);
             },
+            onError: (props) => {
+                // We just want to clear the cache and keep retrying
+                this.memoizedConnections.clear();
```

Comment:
should we only clear cache on `unauthorized` error?

Reply:
mmh good point, maybe the full clear is overkill